src/features/Filters/filter.slice.ts

```diff
@@ -0,0 +1,106 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+interface filterState {
+  activeFilter: boolean;
+  logsByTag: string[];
+  filterByDate: boolean;
+  logsBydate: string[];
+  filterByLocation: boolean;
+  logsByLocation: string[];
+  filterByPic: boolean;
+  logsWithPics: string[];
+  filteredLogsIds: string[];
+  [key: string]: boolean | string[];
```

Comment:
what's this property for? it'd be good to have a comment explaining why `filterState` could take any key
```suggestion
/** some comment */
  [key: string]: boolean | string[];
```